js/tinymce/classes/dom/EventUtils.js

```diff
@@ -138,15 +138,15 @@ define("tinymce/dom/EventUtils", [], function() {
 	function bindOnReady(win, callback, eventUtils) {
 		var doc = win.document, event = {type: 'ready'};
 
-		if (eventUtils.domLoaded) {
+		if (eventUtils.isDomLoaded()) {
 			callback(event);
 			return;
 		}
 
 		// Gets called when the DOM is ready
 		function readyHandler() {
-			if (!eventUtils.domLoaded) {
-				eventUtils.domLoaded = true;
+			if (!eventUtils.isDomLoaded()) {
+				eventUtils.isDomLoaded() = true;
```

Comment:
Is this really supposed to be assigning true to a function call? I would have thought this line would just be removed..?